tests/cases/Mapping/RequestEntityMapping.phpt

```diff
@@ -79,3 +81,27 @@ test(function (): void {
 		$request = $mapping->map($request, $response);
 	}, InvalidStateException::class, sprintf('Attribute "%s" is required', RequestAttributes::ATTR_ENDPOINT));
 });
+
+// Mapping from query or body
+test(function (): void {
+	$request = new ApiRequest(Psr7ServerRequestFactory::fromSuperGlobal());
+	$entity = new NotEmptyEntity();
+
+	$queryRequest = $request
+		->withQueryParams(['foo' => 1]);
+
+	$bodyRequest = $request
+		->withBody(Utils::streamFor(json_encode(['foo' => 1])));
+
+	foreach (['GET', 'DELETE'] as $method) {
```

Comment:
Can you use constants from Endpoint::METHOD_?

Reply:
Sure